Dockerfile.ubi9

```diff
@@ -42,8 +49,7 @@ ARG SKOPEO_BINARY_FILE_SHASUM256=2f00be6ee1c4cbfa7f2452be90a1a2ce88fd92a6d0f6a2e
 ARG ECR_CREDENTIAL_HELPER_VERSION=0.7.1
 ARG ECR_CREDENTIAL_HELPER_BINARY_FILE_SHASUM256=a82cc3ed2cf959616212e3c3c3893dda4f7886da1447c444ef541e6f595ae087
 # https://github.com/chrismellard/docker-credential-acr-env/releases
```

Comment:
nit: remove comment, as we are not referencing a release of this dependency, but building it directly